Sprintfinity3902/Game1.cs

```diff
@@ -169,10 +168,13 @@ public void SetCommands()
             input.RegisterCommand(new SetPlayerMoveCommand(link, link.facingDown), Keys.S, Keys.Down);
             input.RegisterCommand(new SetPlayerMoveCommand(link, link.facingRight), Keys.D, Keys.Right);
             input.RegisterCommand(new SetDamageLinkCommand(this), Keys.E);
- 
+            input.RegisterCommand(new UseBombCommand(link, (BombItem)bombItem), Keys.D1);
+            input.RegisterCommand(new UseBoomerangCommand(link, (BoomerangItem)boomerangItem), Keys.D2);
```

Comment:
Commands might need a rework after Wednesday, Each is very different and this code is kind of confusing.